pages/index.vue

```diff
@@ -159,81 +140,64 @@ export default {
 </script>
 
 <style lang="postcss" scoped>
-.page-home .page-homeLandingFocus,
-.page-home .page-homeLandingFocus::after {
+.landing__background,
+.landing__background::after {
     position: relative;
-    top: 48px;
     left: 0;
     right: 0;
-    height: calc(100vh - 48px);
-    min-height: 720px;
+    height: calc(100vh - 3rem);
+    min-height: 600px;
 }
-
-.page-home .page-homeLandingFocus::after {
+.landing__background::after {
+    height: 100vh;
     position: absolute;
-    top: -48px;
+    top: -72px;
     z-index: -10;
     content: '';
-    background-image: url('~@/static/page-home-background.svg');
-    background-position: center center;
-    background-size: cover;
+    background-image: url('~@/static/landing-background-rwd.svg');
     background-repeat: no-repeat;
+    background-position: center top 4rem;
+    background-size: 100%;
 }
-
-@media (max-width: 567px) {
-    .page-home .page-homeLandingFocus::after {
-        content: '';
-        background-image: url('~@/static/page-home-background-rwd.svg');
-        background-position: center top 10%;
-        background-size: contain;
-        background-repeat: no-repeat;
-    }
-    .page-home__title {
-        left: 50%;
-        margin-left: -50vw;
-        margin-right: -50vw;
-        max-width: 100vw;
-        position: relative;
-        right: 50%;
-        width: 100vw;
-    }
+.landing__title {
+    position: relative;
+    top: 2.5rem;
+    background-image: url('~@/static/landing-title-rwd.svg');
+    background-repeat: no-repeat;
+    background-position-x: center;
+    background-position-y: 2rem;
+    background-size: 100%;
+    min-height: 32vh;
+    min-width: 100%;
 }
-
-.page-home__title {
-    @apply font-serif text-3xl font-semibold;
-    @media (min-width: 1024px) {
-        @apply leading-snug;
-        font-size: 2.8rem;
+@media (min-width: 768px) {
+    .landing__background::after {
+        background-image: url('~@/static/landing-background.svg');
+        background-size: cover;
+        background-position-y: 0;
+    }
+    .landing__title {
+        background-image: url('~@/static/landing-title.svg');
+        background-repeat: no-repeat;
+        background-position-x: 0;
+        background-position-y: 2.5rem;
+        background-size: 60%;
```

Comment:
**issue (blocking):** looks like the title image is not displaying correctly on my devices with screen resolutions of 1920x1080 and 1080x800, as shown in the following images.

**1920x1080**
![image](https://user-images.githubusercontent.com/18432820/206531818-d93ccb4a-a10d-441b-85c6-d5321f4681f9.png)

**1080x800**
<img width="693" alt="image" src="https://user-images.githubusercontent.com/18432820/206531578-d74ef9f5-69b2-43f6-aefa-d8d3b2619f6a.png">